apps/client/src/pages/time-table/page/add-festival.tsx

```diff
@@ -25,26 +28,24 @@ const AddFestival = () => {
         className={styles.headerLayout}
       />
       <div className={styles.container}>
-        {PERFORMANCE_DATA.data.performances.map((performance) => {
-          const isSelected = selectedFestivals.includes(
-            performance.performanceId,
-          );
-
+        {festivals.map((festival) => {
+          const isSelected = selectedFestivals.includes(festival.festivalId);
           return (
             <FestivalCard
-              key={performance.performanceId}
-              id={performance.performanceId}
-              type={performance.type}
-              title={performance.title}
-              imageSrc={performance.posterUrl}
+              key={festival.festivalId}
+              id={festival.festivalId}
```

Comment:
```suggestion
              key={performance.index}
              id={performance.typeId}
```
명세서가 바뀌었습니다 수정해주세요 !